core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/export/localfile/ExportToLocalFile.java

```diff
@@ -83,23 +84,23 @@ public void setKey(final String key) {
 
 
     @Override
-    public final Iterable<String> getInput() {
+    public final Iterable<? extends String> getInput() {
         return input;
     }
 
     @Override
-    public void setInput(final Iterable<String> input) {
+    public void setInput(final Iterable<? extends String> input) {
         this.input = input;
     }
 
     @Override
-    public TypeReference<Iterable<String>> getOutputTypeReference() {
-        return null;
+    public TypeReference<Iterable<? extends String>> getOutputTypeReference() {
+        return new TypeReferenceImpl.IterableString();
```

Comment:
We can use `IterableString` directly.
```suggestion
    public IterableString getOutputTypeReference() {
        return new IterableString();
```